packages/wdio-browserstack-service/src/util.ts

```diff
@@ -21,10 +24,11 @@ export function getBrowserDescription(cap) {
 
 /**
  * get correct browser capabilities object in both multiremote and normal setups
- * @param {*} caps browser capbilities object. In case of multiremote, the object itself should have a property named 'capabilities'
- * @param {*} browserName browser name in case of multiremote
+ * @param browser browser object
+ * @param caps browser capbilities object. In case of multiremote, the object itself should have a property named 'capabilities'
+ * @param browserName browser name in case of multiremote
  */
-export function getBrowserCapabilities(browser, caps, browserName) {
+export function getBrowserCapabilities(browser: Browser, caps: Capabilities = {}, browserName: string = '') {
```

Comment:
We should not default to empty string
```suggestion
export function getBrowserCapabilities(
    browser: Browser,
    caps: Capabilities = {},
    browserName?: string
) {
```

Reply:
Done and fixed types